internal/validate/validate_test.go

```diff
@@ -0,0 +1,53 @@
+package validate_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/ibc-go/v8/internal/validate"
+)
+
+func TestGRPCRequest(t *testing.T) {
+	const (
+		validID   = "validIdentifier"
+		invalidID = ""
+	)
+	testCases := []struct {
+		msg       string
+		portID    string
+		channelID string
+		expPass   bool
```

Comment:
I wanted to test for specific error types rather than a boolean presence, but I ran into some issues there and also I don't know how helpful it is since this function only returns either no error or the same type of error in both branches.

Reply:
I am generally not a fan of testing error messages, as it can be seen as a change-detector test (https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html) which is usually not useful. But I'm open to changing my mind :)